examples/connext_dds/flat_data_latency/c++11/Common.hpp

```diff
@@ -170,3 +129,13 @@ inline void configure_nic(
                 nic});
     }
 }
+
+inline void print_latency(const int &total_latency, const int &count)
```

Comment:
Added print_latency function

Reply:
Pass arguments by value (they are primitives): print_latency(int, int)

Reply:
See rules F.15 and F.16 in https://bitbucket.rti.com/projects/CONNEXT/repos/cpp_core_guidelines/browse/CppCoreGuidelines.md#